File: coko/com.elster.jupiter.metering.extjs/src/main/web/js/mtr/src/readingtypes/attributes/store/Status.js
/*
 * Copyright (c) 2017 by Honeywell International Inc. All Rights Reserved
 */

Ext.define('Mtr.readingtypes.attributes.store.Status',{
    extend: 'Ext.data.Store',
    autoLoad: false,
    fields: ['code', 'displayName'],
    data : [
        {code: 'true', displayName: Uni.I18n.translate('readingtypesmanagment.active', 'MTR', 'Active')},
        {code: 'false', displayName: Uni.I18n.translate('readingtypesmanagment.inactive', 'MTR', 'Inactive')}
    ]
});

File: coko/com.elster.jupiter.calendar/src/main/java/com/elster/jupiter/calendar/Status.java
/*
 * Copyright (c) 2017 by Honeywell International Inc. All Rights Reserved
 */

package com.elster.jupiter.calendar;

import com.elster.jupiter.nls.Thesaurus;

public enum Status {
    ACTIVE, INACTIVE;

    public String getDisplayName(Thesaurus thesaurus) {
        return CalendarStatusTranslationKeys.translationFor(this, thesaurus);
    }
}
